backend/manager/modules/vdsbroker/src/main/java/org/ovirt/engine/core/vdsbroker/VmManager.java

```diff
@@ -169,6 +171,8 @@ public Lock getVmDevicesLock() {
 
     public void update(VmDynamic dynamic) {
         vmDynamicDao.update(dynamic);
+        numOfCores = cpuPinningPolicy == CpuPinningPolicy.RESIZE_AND_PIN_NUMA ? 0
```

Comment:
can you pull it and the above (static) to a method? something like:
```
private int getDbNumOfCores(boolean static) {
    if (cpuPinningPolicy == CpuPinningPolicy.RESIZE_AND_PIN_NUMA) {
        return 0;
    }
    return static ?  vmStatic.getNumOfCpus(false) : dynamic.getCurrentSockets() * dynamic.getCurrentCoresPerSocket();
}
```

Reply:
There was a bug in that part of the code in the previous commit and now those two parts are different. Resolving as not relevant any more.